52.md

```diff
@@ -80,6 +104,14 @@ Example:
   ]
 }
 ```
+#### Private Event
+
+It's an _addressable event_ of `kind:32677`.
+Apart from the public event tags, there is one more tag:
+* `description` Description of the event
+
+The corresponding gift wrap is of `kind:1052`
```

Comment:
Are we creating new wrap kinds? Then don't call it "gift" wrap. There is only one gift wrap kind.

Reply:
I used the term "gift wrap" to easily explain the process. Will update the term. However the wrapping process is the same as gift wrap